synth.py

```diff
@@ -21,10 +21,14 @@
 
 s.replace(".kokoro/build.sh", """(export PROJECT_ID=.*)""", """\g<1>
 
-# Configure Local Redis to be used
+# Configure local Redis to be used
 export REDIS_CACHE_URL=redis://localhost
 redis-server &
 
+# Configure local memcached to be used
+export MEMCACHED_HOSTS=localhost
```

Comment:
Change `localhost` to `127.0.0.1`, and rerun synth to pick up the change in `.kokoro/build.sh`. It seems what was happening was the client's notion of `localhost` changed from the ipv4 version to the ipv6 version for some reason, but memcached is listening on ipv4 `127.0.0.1`. This seems to fix that.

Reply:
@chrisrossi shouldn't we do the same for the redis URL? In part to prevent that happening there and in part to be consistent?

Reply:
Or if it's not broken why fix it?

Reply:
It worked! Leaving this question for another time.